src/gordon_gcp/plugins/janitor/reconciler.py

```diff
@@ -240,6 +240,33 @@ def _parse_rrset_message(self, message):
 
         return zone, rrsets
 
+    @staticmethod
```

Comment:
what's the reasoning for using a static method here?

Reply:
Well, it doesn't need anything from the instance, so...

Reply:
hm, maybe i am missing something - but since it's usage from `validate_rrsets_by_zone` is in the class anyway, why not just call it by doing `_remove_soa_and_root_ns`?

Reply:
The call would be either GDNSReconciler._remove_soa_and_root_ns (static method) or self._remove_soa_and_root_ns (instance method).  It's not strictly required to make this a static method, but since it doesn't need anything from the instance, that's what I generally do.  That way it can, in theory, be used by something else without instantiating the class.  If you think it's a bad idea I can change it.

Reply:
Oh this is interesting.  Turns out the decorator makes it possible to call the method from the instance, and isn't necessary otherwise.  So I could change the calls without changing the method.  I do like to make it clear that it's a static method in how I call it though.

Reply:
@smstone pointed out that it's also private.  Meh, made it instance.